src/devhub/components/organism/Navbar.jsx

```diff
@@ -243,15 +243,27 @@ return (
                 {link.title}
               </MobileLink>
             ) : (
-              link.links.map((it, idx) => (
-                <MobileLink
-                  key={`nested-link-${idx}`}
-                  className={link.href === props.page && "active"}
-                  href={`/${REPL_DEVHUB}/widget/app?page=${it.href}`}
-                >
-                  {it.title}
-                </MobileLink>
-              ))
+              link.links.map((it, idx) =>
+                it.href.startsWith("http://") ||
+                it.href.startsWith("https://") ? (
+                  <MobileLink
+                    key={`nested-link-${idx}`}
+                    className={link.href === props.page && "active"}
```

Comment:
External links cannot be "active", so this condition is useless, but I will merge it as a hotfix and leave the whole block for refactoring after NEARCON.